src/ReturnTypes/AuthExtension.php

```diff
@@ -31,7 +32,7 @@ public function getClass(): string
 
     public function isStaticMethodSupported(MethodReflection $methodReflection): bool
     {
-        return $methodReflection->getName() === 'user';
+        return in_array($methodReflection->getName(), ['user', 'authenticate']);
```

Comment:
Just a nit, but I generally prefer to pass `strict: true`/`true` to my `in_array` calls (`===` instead of `==` equivalent)---up to you though


```suggestion
        return in_array($methodReflection->getName(), ['user', 'authenticate'], strict: true);
```